### DISCUSSION TITLE
Contributor call: 11 June 2022

### DISCUSSION THREAD

**Discussion by l3xander** [freesewing/freesewing#2240] [Support] [answered]
You are all invited to the FreeSewing Contributor Call.
Please leave a comment here or [on Discord](https://discord.freesewing.org/) to add things to the agenda.
This issue will be updated after the call with notes and to-dos.

### About
FreeSewing's contributor call is an informal hangout/chat/meeting that is open to all FreeSewing contributors and users alike.
The call is organized/hosted by @tangerineshark.
The call does not have a fixed structure or agenda items. Everyone is free to suggest agenda items, ask questions, or bring ideas.

Details:
Location: https://discord.freesewing.org (meeting room: 🔊voice chat)
Date: 11 June 2022 4pm UTC, which can be found in your local time zone [here](https://savvytime.com/converter/utc-to-belgium-antwerpen-ca-los-angeles-il-chicago-ny-new-york-city-brazil-sao-paulo-united-kingdom-london-france-paris-south-africa-johannesburg-msd-india-mumbai-hkt-japan-tokyo-australia-sydney/jun-11-2022/4pm).
Or check the bot in Discord! #meetings
The meetings will always be at 6pm, Antwerp time.

### Agenda
- Jacqueline's feedback

> This discussion will be updated after the call with notes and to-dos.

**Comment by tangerineshark** [comment, marked as answer]
### Notes

- @tangerineshark wanted to crowdsource thoughts about newsletter additions for the 1 July newsletter while there's still a touch of time, suggestions listed below:
    - There's a bunch of work going on behind the scenes on next.freesewing.org - it would be cool to hear about both the new design features and the updates to the backend if it's not too soon for that. We realize though that this almost has to be an ask of @joostdecock , so no pressure. If we decide to do this, maybe someone with UI/UX background could help write about the design updates? Related but not a to do: there was much raving about the new Showcase, and how great it looks.
    - Could post a small announcement about @lexander2002 's upcoming talk?
    - @nataliasayang mentioned that a few users have recently posted patterns adapted to children, and tips and notes on their approach to this might be helpful. In particular, comixminx has adapted several patterns for youth, and this discord comment discusses adapting a collared button-up for a toddler: https://discord.com/channels/698854858052075530/757631205804998759/971763891182899240 
    - Current pattern releases: Lucy pocket pattern by @SeaZeeZee
    - Maybe something about Hi by @woutervdub, but it might not be out yet? Lots of enthusiasm for a little article on how you approach drafting something like this (completely separate from human measurements) in the FreeSewing environment
- Should @tangerineshark try to make a pirate shirt? There's been a bit of enthusiasm for them in the chat of late, and they're basically a bunch of rectangles... @SeaZeeZee is also considering making one of these - could collaborate
- @bobgeorgethe3rd gave a tour of `Theobald`, a reimagining of Theo based on Titan. This could swap in for Theo once it's deprecated in v3. It's pretty awesome looking, and has some new bonuses, like a few different pocket options and a fishtail waistband, heavily encouraged by the aquatic among us
- Discussion of Teagan shaping: a discord post mentioned weird shaping on the side seam of Teagan, where it's sort of bell shaped from armscye to hem. @nataliasayang and @tangerineshark also have that bowing out of the side seam to different degrees, and wonder if it could also go straight down to the hip, taper in, etc.? Also curious how the waist measurement is used for Teagan and if that's related?

*Note from @tangerineshark, who wrote these notes: please comment or edit with clarifications or questions if I forgot anything!!* 😅

**Comment by joostdecock** [comment]
Closing to increase visibility of active discussions
